.github/workflows/tag.yml

```diff
@@ -61,3 +61,19 @@ jobs:
             golangci/golangci-lint:${{ steps.prepare.outputs.tag_name }}-alpine
             golangci/golangci-lint:${{ steps.prepare.outputs.major_tag }}-alpine
             golangci/golangci-lint:latest-alpine
+  update_github_action:
```

Comment:
the related make target will make changes in respective file, we need to add the step to check in changes as well.

Reply:
GITHUB_TOKEN is already there, you can use checkout@v2 as mentioned in this [link](https://github.community/t/how-does-one-commit-from-an-action/16127/6)

Reply:
Another point is about pushing to master
- pushing to master might not be allowed
- even it's allowed, it's kind of risky :thinking: 

So automate this work might be just sending PR (like dependabot), and let other member to review and merge.

Reply:
@sayboras Thanks for reviewing this MR. I have a few questions to clarify

1. Do you propose I save the changed files as artifacts?
2. Should I move these into another workflow after the release?
3. Can we create a dummy release and test?

Reply:
> Do you propose I save the changed files as artifacts?
I am thinking about sending changed files in another PR _automatically_

> Should I move these into another workflow after the release?
We can keep it in same workflow, but only run it if the gorelease + docker jobs completed successfully (e.g. use `need` attribute in github action)

> Can we create a dummy release and test?
I normally test this kind of changes in forked repo, in which you will have _full permission_. Then just share details of your testing here.

Hope it's clear, feel free to slack me in gopher channel #golangci-lint as well :+1: